### PR TITLE
Redesign_Bbaker: Round display Component 

### PR DESCRIPTION
**Pull request by BBtxt**
Redesigned the round display to use shadui card component. 

- Only displays past rounds with link to Sound Cloud playlist and link to round details 
- style for responsive design.

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -19,11 +19,11 @@ const Homepage = async () => {
       <Head>
         <title>Home | Everyone Plays the Same Song</title>
       </Head>
-      <div className="flex justify-center items-center h-4/5">
+      {/* <div className="flex justify-center items-center h-4/5"> */}
```
Comment (nspilman) [inline app/page.tsx:22]: you can just delete these. Any reason to keep the code?

File: app/layout.tsx [hunk 2]
```diff
@@ -19,7 +20,8 @@ export default async function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body>
+      <body className={cn(
```
Comment (nspilman) [inline app/layout.tsx:23]: is the `cn` util function required here? would just passing the className not be sufficient?

**Review by nspilman — APPROVED** [review]
Approved with smol questions/comments

File: app/layout.tsx (hunk 2 above)
Comment (BBtxt) [reply app/layout.tsx:23]: I'll look deeper into this but, in the shad ui docs this seemed to be required. Once added I was able to see the old components.

File: app/page.tsx (hunk 1 above)
Comment (BBtxt) [reply app/page.tsx:22]: Cleaned up in last commit

**Pull request merged**